src/Nethermind/Nethermind.Api/IApiWithBlockchain.cs

```diff
@@ -74,7 +74,7 @@ public interface IApiWithBlockchain : IApiWithStores, IBlockchainBridgeFactory
         IHealthHintService? HealthHintService { get; set; }
         IRpcCapabilitiesProvider? RpcCapabilitiesProvider { get; set; }
         ITransactionComparerProvider? TransactionComparerProvider { get; set; }
-        ITxValidator? TxValidator { get; set; }
+        TxValidator? TxValidator { get; set; }
```

Comment:
I would like this to remain an interface. Any reason not to?

Reply:
Required due to the change on `NethermindApi`